spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/main/java/com/example/Application.java

```diff
@@ -25,10 +25,12 @@
 *
 * @author Jisha Abubaker
 */
+//HideUtilityClassConstructorCheck OFF
```

Comment:
Good point. 
@meltsufin what do you think?

Reply:
If we excluded samples, then they would have no checks run on them at all, which I find a little risky.
The suppression is ugly too.
Is it possible to modify the rule upstream to recognize that this is not a utility class because it has a `main` method?

Reply:
@dmitry-s Is it possible to just exclude it for the specific check? Maybe we need an exclusions checkstyle.xml or something?

Reply:
See `checkstyle-suppressions.xml` for exclusion this or that.

In Spring Integration we have it like this:
```
<?xml version="1.0"?>
<!DOCTYPE suppressions PUBLIC
    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
    "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
<suppressions>
	<suppress files="package-info\.java" checks=".*" />
	<suppress files="[\\/]test[\\/]" checks="RequireThis" />
	<suppress files="[\\/]test[\\/]" checks="FinalClass" />
	<suppress files="[\\/]test[\\/]" checks="InnerTypeLast" />
	<suppress files="CachingSessionFactory" checks="FinalClass" /> <!-- Tests spy -->
	<suppress files="[\\/]test[\\/]" checks="Javadoc*" />
	<suppress files="[\\/]test[\\/]" checks="ImportOrder" />
	<suppress files="[\\/]test[\\/]" checks="EqualsHashCode" />
</suppressions>
```